packages/sitecore-jss-react/src/components/Image.tsx

```diff
@@ -143,7 +143,7 @@ export const getEEMarkup = (
     return getEditableWrapper(imageField.editable);
   }
 
-  const imgHtml = ReactDOMServer.renderToStaticMarkup(<img {...imgAttrs} />);
+  const imgHtml = `<img ${getAttributesString(imgAttrs as { [key: string]: string })} />`;
```

Comment:
Here you set type _{ [key: string]: string }_. Is it only limited to string value? I see that _getAttributesString_ supports more value types